xray/cli.go

```diff
@@ -274,6 +285,23 @@ func createServerDetailsWithConfigOffer(c *cli.Context) (*coreconfig.ServerDetai
 }
 
 func shouldIncludeVulnerabilities(c *cli.Context) bool {
-	// If no contex was provided by the user, no Violations will be triggered by Xray, so include general vulnerabilities in the command output
+	// If no context was provided by the user, no Violations will be triggered by Xray, so include general vulnerabilities in the command output
 	return c.String("watches") == "" && c.String("project") == "" && c.String("repo-path") == ""
 }
+
+func validateXrayContex(c *cli.Context) error {
+	contextFound := false
+	contextErorrMsg := "only one of the following flags can be supplied: --watches,--project or --repo-path."
```

Comment:
Missing white-space after ```--watches,```. Should be:
...supplied: --watches, --project or --repo-path.